ProcessLib/BoundaryCondition/RobinBoundaryConditionLocalAssembler.h

```diff
@@ -64,17 +65,29 @@ class RobinBoundaryConditionLocalAssembler final
             _data.u_0.getNodalValuesOnElement(Base::_element, t)
                 .template topRows<ShapeFunction::MeshElement::n_all_nodes>();
 
+        ParameterLib::SpatialPosition position;
+        position.setElementID(Base::_element.getID());
+
         for (unsigned ip = 0; ip < n_integration_points; ++ip)
         {
+            position.setIntegrationPoint(ip);
             auto const& ip_data = Base::_ns_and_weights[ip];
             auto const& N = ip_data.N;
             auto const& w = ip_data.weight;
 
+            double integral_measure = 1.0;
+            if (_data.integral_measure)
+            {
+                integral_measure = (*_data.integral_measure)(t, position)[0];
```

Comment:
```suggestion
                const double integral_measure =   (_data.integral_measure) ? (*_data.integral_measure)(t, position)[0] : 1.0;
```

Reply:
I can't declare/define the variable `integral_measure` only in the scope of the branch of the if condition since it is used below.

Reply:
I see.

Reply:
:fast_forward: